a/apache-flink/apache-flink_master_rhel_8.3.sh

```diff
@@ -18,23 +18,68 @@ VERSION="master"
 
 # Install dependencies and tools.
 sudo yum update -y
-sudo yum install -y git wget java-1.8.0-openjdk-devel
+sudo yum install -y git wget java-1.8.0-openjdk-devel patch xz
 export JAVA_HOME=export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
 
-#Install maven v3.2.5 which is recommended by Flink 1.8.x
-wget https://www-us.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz --no-check-certificate --quiet
-tar xzf apache-maven-3.2.5-bin.tar.gz
+# Install postgresql build dependencies
+sudo yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-ppc64le/pgdg-redhat-repo-latest.noarch.rpm
+sudo yum install -y postgresql10-libs
+sudo yum install -y postgresql10 postgresql10-server postgresql10-contrib
 
-export M2_HOME=`pwd`/apache-maven-3.2.5
-export PATH=`pwd`/apache-maven-3.2.5/bin:${PATH}
+# Install maven v3.6.3
```

Comment:
Should we make the maven version configurable?

Reply:
Created ENV variable to maintain the maven version.